dp0-delegate-resources/index.rst

```diff
@@ -22,37 +22,61 @@ DP0 Delegate Homepage
 
 This page includes the guidelines, resources, and activities for DP0 Delegates.
 
-**DP0 delegate** is the term for the 600 scientists and students who have accounts in the Rubin Science Platform (RSP)
+"**DP0 delegates**" are the scientists and students who have accounts in the Rubin Science Platform (RSP)
 at the Interim Data Facility (IDF; the Google Cloud) during Data Preview 0 (DP0).
-This term has been adopted to reflect DP0 participants’ important roles of representing the broad science community
+
+The term "delegate" has been adopted for DP0 to reflect participants’ important roles of representing the broad science community
 as learners, testers, and providers of feedback, and of sharing the benefits of their DP0 participation with their
 communities as teachers and colleagues.
-Delegates’ DP0-related activities are supported with resources and infrastructure described on this page.
+Delegates’ DP0-related activities are supported with resources and infrastructure as described on this page.
+
+The number of delegates is currently limited to 600, but space is available.
 
 
 .. _Delegate-Homepage-Getting-Started-Checklist:
 
 Getting started with DP0 checklist
 ==================================
 
+Becoming a delegate is as easy as following this checklist and getting an account in the Rubin Science Platform (RSP).
```

Comment:
"Becoming a delegate and accessing the DP0"

Reply:
incorporated this wording too